release/release_gold.sh

```diff
@@ -170,6 +171,11 @@ function tag_release {
 		repository=liferay-portal
 	fi
 
+	if [ -z "${LIFERAY_RELEASE_REPOSITORY_OWNER}" ]
+	then
+		LIFERAY_RELEASE_REPOSITORY_OWNER=liferay
+	fi
+
```

Comment:
@lucasmiranda0, can you please move this code to be above the declaration of the variable `repository`?